### DISCUSSION TITLE
invisible text layer

### DISCUSSION THREAD

**Discussion by lbr991** [py-pdf/fpdf2#838] [Q&A]
I'm trying to take a non-searchable pdf and convert it to searchable pdf by superimposing an invisible text layer (or similar).
I need the invisible text bboxes to align exactly with the original bboxes (extracted by Textract).
I also don't want to specify a font and font_size because then the bboxes wouldn't align perfectly.

Is this possible with fpdf2?

**Comment by Lucas-C** [comment]
Hi @lbr991

> Is this possible with fpdf2?

I do not think so, sorry.

Have you checked this section of our documentation?
https://pyfpdf.github.io/fpdf2/CombineWithPdfrw.html

Also, do you know this simple trick to use a browser in order to convert a protected PDF to an unprotected PDF?
https://pdf.wondershare.com/how-to/copy-secured-pdf.html
